src/sdk/namespace_gcp.js

```diff
@@ -234,7 +234,16 @@ class NamespaceGCP {
     async delete_multiple_objects(params, object_sdk) {
         // https://googleapis.dev/nodejs/storage/latest/File.html#delete
         dbg.log0('NamespaceGCP.delete_multiple_objects:', this.bucket, inspect(params));
-        throw new S3Error(S3Error.NotImplemented);
+
+        const res = await P.map_with_concurrency(10, params.objects, obj =>
```

Comment:
GCS seems to support [batch requests](https://cloud.google.com/storage/docs/batch), but it is not part of the js SDK. Maybe it's worth investing some time in working with it directly using REST API and not using the SDK (we might be able to take auth info from the SDK).  

batch requests will be very useful for many operations in the google NS. Specifically in this case it can reduce the number of requests significantly (`delete_multiple_objects` can contain up to 1000 objects)